proto/storage/node.proto

```diff
@@ -86,10 +89,18 @@ message NodeScan {
 }
 
 message NodeInventory {
-    string                         node_id    = 1 [(gogoproto.moretags) = 'search:"Node ID,store" sql:"pk,type(uuid)"'];
-    string                         node_name  = 2 [(gogoproto.moretags) = 'search:"Node,store"'];
-    google.protobuf.Timestamp      scan_time  = 3 [(gogoproto.moretags) = 'search:"Node Scan Time,store"'];
-    scannerV1.Components           components = 4;
+    reserved 1; // was node_id; now node_name is used to define the relation between node_inventory with a node
```

Comment:
Question: We removed the `node_id` here, because `NodeInventory` is now always sent as part of a `Node` message, which already contains the `ID`, correct?

Reply:
Yes. If we would keep it here, the info would be duplicated